### ISSUE TITLE
navigation error when selecting NWB file

### ISSUE THREAD

**Issue by bendichter** [scratchrealm/neurobass#13]
If you select an NWB file, it becomes impossible to navigate back to the DANDI search screen

https://github.com/scratchrealm/neurobass/assets/844306/82a8889e-6174-4b92-bdad-69dde618e83a

**Comment by magland** [comment]
See the "DANDI Import" button at the top of that window.

I guess there should be another more obvious button "Back to DANDI"

**Comment by bendichter** [comment]
Yes I think it would help if these were tabs as opposed to radio dials (though I suppose this type of UX fine-tuning can wait)
